docs/whats-new.mdx

```diff
@@ -14,6 +14,7 @@ You can expect this page to update regularly (at least monthly). We'll include t
 
 ## June 2024
 
+- 2024-06-13 - Agent now has timestamps for incoming `http` requests.
```

Comment:
For this one we want to mention the newly released agent version and link to the changelog which has the information on what has actually changed.

Reply:
Thanks! I pulled in your chagnes here: https://github.com/ngrok/ngrok-docs/pull/788/commits/4140ed4ac41d99551ecffd0433d972ab5bb407cd